settings.ui

```diff
@@ -205,6 +205,39 @@
          </layout>
         </widget>
        </item>
+       <item>
+        <widget class="QGroupBox" name="eepromGroupBox">
+         <property name="title">
+          <string>EEPROM</string>
+         </property>
+         <layout class="QGridLayout" name="eepromGridLayout">
+          <item row="0" column="0">
+           <widget class="QLabel" name="eepromImageLabel">
```

Comment:
nit: For consistency, this should be `eepromPathLabel` or the below names should be `eepromImage` or `eepromImagePath`

Reply:
Changed to eepromPathLabel